### ISSUE TITLE
Compiler error to swift 4

### ISSUE THREAD

**Issue by danielideriba** [Johnykutty/JKSteppedProgressBar#8]
Hi 
I figured out a strange issue when I try to build application:
Pods/JKSteppedProgressBar/JKSteppedProgressBar/Classes/SteppedProgressBar.swift:161:27: 'NSForegroundColorAttributeName' has been renamed to 'NSAttributedStringKey.foregroundColor' 

I had to set up on podfile: "pod 'JKSteppedProgressBar'", and i'm using swift 4.0 in project.
Do you have any ideias to solve this issue?

**Comment by mhausherr** [comment]
Temporary solution for your build : in your Podfile add these few lines

```
post_install do |installer|
    installer.pods_project.targets.each do |target|
        if ['JKSteppedProgressBar'].include? target.name
            target.build_configurations.each do |config|
                config.build_settings['SWIFT_VERSION'] = '3.2'
            end
        end
    end
end
```

**Comment by jkmathew** [comment]
Fixed in #9
